graphql_compiler/query_pagination/pagination_planning.py

```diff
@@ -136,7 +136,7 @@ def get_pagination_plan(
     # Get the pagination capacity
     vertex_path = (root_node,)
     property_path = PropertyPath(vertex_path, pagination_field)
-    capacity = query_analysis.pagination_capacities.get(property_path)
+    capacity = query_analysis._pagination_capacities.get(property_path)
```

Comment:
This usage of private field outside of the class can be removed by making pagination_planning one of the analysis passes, just as we did with cost estimation.